docs/Deploying/TUF-Update-Guide.md

```diff
@@ -0,0 +1,188 @@
+# TUF Update Guide
+
+This document is a walkthrough guide for how a Fleet member can become a publisher of updates for [Fleet's TUF service](tuf.fleetctl.com).
+The roles needed to push new updates are `targets`, `snapshot` and `timestamp`.
+
+## Security
+
+- TUF keys for `targets`, `snapshot` and `timestamp` should be stored on a USB stick.
+- The keys are stored encrypted with a passphrase stored in 1Password.
+
+## Sync Fleet's TUF repository
+
+For simplicity we sync the repository to the USB stick.
+```sh
+cd /Volumes/FLEET-TUF/repository
+
+cd /Volumes/FLEET-TUF
+mkdir -p repository
+mkdir -p keys
+mkdir -p staged
+
+aws s3 sync s3://fleet-tuf-repo ./repository
+```
+
+## Generate targets+snapshot+timestamp keys
+
+```sh
+tuf gen-key targets
```

Comment:
These keys should be generated in the `/Volumes/FLEET-TUF/keys` directory, right?

Reply:
Correct, `tuf` creates them in `$(pwd)/keys`. I'll add a comment that all `fleetctl` and `tuf` commands are run from `/Volumes/FLEET-TUF`